content/ytt/docs/latest/ytt-overlays.md

```diff
@@ -33,10 +33,11 @@ Given a sample target YAML file:
 > #@overlay/match by=overlay.all, expects="1+"
 > ---
 > contents:
-> #@overlay/append
 > - pen
 > ```
 
+Prior to v0.32.0 append array items with [`#@overlay/append`](lang-ref-ytt-overlay.md#@overlay/append)
```

Comment:
Same as earlier comment. This doesn't direct you to the correct anchor as expected.

Reply:
Unrelated question about this document, how can you navigate to it? I didn't see it listed in the sidebar nav so I'm curious how someone finds this. I struggled to find it while doing this review.

Reply:
@aaronshurley The link anchors have been corrected. The ytt-overlays doc, titled "Using overlays", has been added to the left side bar navigation in the ytt docs page. It was placed as the second item in the "Techniques" section, right after "using data values". Let us know with a 👍  if this looks good.